.github/workflows/publish-tags.yaml

```diff
@@ -1,18 +1,21 @@
-name: Publish Docker - Release
+name: Publish to Registry - Release
 on:
+  release:
+    types: [published]
   push:
-    tags:
-      - v*
+    branches:
+      - master
+  schedule:
+    - cron: "0 2 * * 0" # Weekly on Sundays at 02:00
 jobs:
-  build:
+  update:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@master
       - name: Publish to Registry
         uses: elgohr/Publish-Docker-Github-Action@master
         with:
-          name: tremorproject/tremor
+          name: myDocker/repository
```

Comment:
Does this refer to some other config value, like a secret or is this a copy paste artefact?

Reply:
already changed :)